.github/workflows/pkg.pr.new-comment.yml

```diff
@@ -0,0 +1,38 @@
+name: Update pkg.pr.new comment
+
+# This workflow is used to comment a pull request with a package created in pkg.pr.new.yml.
+#
+# The GITHUB_TOKEN for the workflow from a pull request
+# does not have the permission to comment on the pull request,
+# so the workflow is split.
+
+on:
+  workflow_run:
+    workflows: [Publish to pkg.pr.new]
```

Comment:
This workflow runs only after a merge into the default branch.

After merging, the next PR will be run and a comment will be added to the PR via scripts/pkg.pr.new-custom-comment.mjs.